.clang-format

```diff
@@ -66,7 +66,7 @@ IndentGotoLabels: true
 IndentPPDirectives: None
 IndentWidth:     4
 IndentWrappedFunctionNames: false
-InsertBraces: true
+#InsertBraces: true
```

Comment:
Is this important?

Reply:
this was just to fix an issue with my local clang version.